reputation_cron.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
We should make a `cron` folder to keep these in one place, since it's a non-obvious part of setting up on a new server that scheduled jobs need to be set up in server config to run scripts like this